integration/common/openlineage/common/provider/dbt/processor.py

```diff
@@ -123,8 +125,10 @@ class ParentRunMetadata:
     job_name: str = attr.ib()
     job_namespace: str = attr.ib()
 
-    def to_openlineage(self) -> ParentRunFacet:
-        return ParentRunFacet.create(runId=self.run_id, name=self.job_name, namespace=self.job_namespace)
+    def to_openlineage(self) -> parent_run.ParentRunFacet:
+        return parent_run.ParentRunFacet.create(
```

Comment:
nit: I think `parent_run.ParentRunFacet.create()` is deprecated? I saw it in multiple places, just assuming we are ignoring this deprecation for now.

Reply:
Yes, I should have remove the usage of it. Just did.